botocore/handlers.py

```diff
@@ -109,6 +110,24 @@ def decode_console_output(parsed, **kwargs):
             logger.debug('Error decoding base64', exc_info=True)
 
 
+def generate_idempotent_uuid(params, model, **kwargs):
+    input_shape = model.input_shape
+    if not input_shape or not hasattr(input_shape, 'members'):
+        return
+    # Might be a mock model from a test that doesn't care about this
+    if not hasattr(input_shape.members.items(), '__iter__'):
```

Comment:
Why would all tests need that? This should only be running in functional / integration tests aside from the unit tests directly related to it.